client/components/NavBar/navBar.scss

```diff
@@ -12,6 +12,17 @@
 		user-select: none;
 		cursor: pointer;
 		text-decoration: none;
+		&:hover {
+			.title {
+				text-decoration: underline;
+			}
+		}
+		&:focus {
+			.title {
+				text-decoration: underline;
+				text-decoration-style: dotted;
+			}
+		}
```

Comment:
What are you thoughts on something like this? To me it reads a bit simpler:

```css
&:hover, &:focus {
  .title {
    text-decoration: underline;
  }
}
&:focus .title {
  text-decoration-style: dotted;
}
```

Reply:
++